lib/cli/src/commands/run.rs

```diff
@@ -224,7 +224,9 @@ impl Run {
         let module_result: Result<Module> = if !self.disable_cache && contents.len() > 0x1000 {
             self.get_module_from_cache(&store, &contents, &engine_type, &compiler_type)
         } else {
-            Module::new(&store, &contents).map_err(|e| e.into())
+            let module = Module::new(&store, &contents)?;
+            let _serialized = module.serialize()?;
```

Comment:
Why running `serialize` here?

Reply:
This was for debugging! I just reverted that commit!